tests/misc/CachingTest.py

```diff
@@ -122,20 +122,28 @@ def test_caching_results(self):
             retval, stdout, stderr = execute_coala(
                 coala.main,
                 'coala',
+                '--non-interactive', '--no-color',
```

Comment:
Not strictly necessary, but it will make it easier to read any logging output on the CI log, at least it will when https://github.com/coala/coala/pull/4224 is fixed.